### DISCUSSION TITLE
Support Pimcore CMS

### DISCUSSION THREAD

**Discussion by 80982proton** [ddev/ddev#5840] [Ideas]
Hello folks,

it would be great if you could add support for Pimcore. @dpfaffenbauer @brusch

Best regards

**Comment by rfay** [comment]
Welcome Pimcore!

I think Pimcore should already work fine with this, could you please check it out?

```bash
#This uses the pimcore skeleton repo:

mkdir my-pimcore && cd my-pimcore
ddev composer create pimcore/skeleton
ddev config --docroot=public --create-docroot
ddev exec pimcore-install --mysql-username=db --mysql-password=db --mysql-host-socket=db --mysql-database=db --admin-password=admin --admin-username=admin --no-interaction
ddev launch
```

If you like that, or have improvements to it that you think are important, we can easily get it into the quickstarts, https://ddev.readthedocs.io/en/latest/users/quickstart (You can add it yourself, or with the additions you think are important) by just clicking the pencil icon in the upper right of that page. 

Welcome, and looking forward to collaboration! We rely on the many communities that use DDEV to make sure that it works right for them.

**Reply by rfay** [reply]
Moved this to an issue, https://github.com/ddev/ddev/issues/5843 - let's continue there.

Looking forward to it!
